newdle/client/src/actions.js

```diff
@@ -27,3 +30,15 @@ export function loadUser() {
 export function tokenExpired() {
   return {type: TOKEN_EXPIRED};
 }
+
+export function setActiveDate(date) {
+  return {type: SET_ACTIVE_DATE, date};
+}
+
+export function setCalendarDate(dates) {
```

Comment:
Isn't this implicit (ie something for a selector) based on which dates have time slots?

Reply:
I wasn't aware of the specific data model thus I assumed we will do it somehow manually.

Reply:
from the notes in the codimd ;)

> Calendar highlights days which have time slots - removing the last time slot removes the whole (empty) day;

and in the issue:

> Days that have time slots should be highlighted so it's clearly visible which days are used. Removing the last time slot from a day should remove this highlight.

By not having this data in the store the logic remains easier since we just need a selector that returns the dates which have time slots

Reply:
Hm, then for now I will just create a selector that will return an empty array. And as soon as we add more stuff we will adapt it.

Reply:
or some random dates in the current month so it looks better until we have it?

Reply:
Yeah, fine as well ;)